Python/Easy/[424] Return the Factorial [E].py

```diff
@@ -84,8 +84,8 @@ def factorial1(n):
             return output
         elif not isinstance(n, int):
             return "Integer input is only valid."
-    elif n < 0:
-        return "Input cannot be less then 0."
+    
+    return "Input cannot be less then 0."
```

Comment:
This logic of your's is flawed, here.
If n<0 then the statement should be returned, not just for it to be returned.